parlai/core/torch_agent.py

```diff
@@ -1671,6 +1674,11 @@ def reset_metrics(self):
 
     def act(self):
         """Call batch_act with the singleton batch."""
+        if self._replies_are_shared:
```

Comment:
why are we raising a runtime error here??? 

am i misunderstanding something, or will this raise a runtime error every time we call act on an interactive agent

Reply:
nevermind, ignore this comment lol